maat-scheduled-tasks/src/main/java/uk/gov/justice/laa/maat/scheduled/tasks/mapper/RepOrderBillingMapper.java

```diff
@@ -1,16 +1,16 @@
 package uk.gov.justice.laa.maat.scheduled.tasks.mapper;
 
 import lombok.extern.slf4j.Slf4j;
-import org.springframework.util.StringUtils;
 import uk.gov.justice.laa.maat.scheduled.tasks.dto.RepOrderBillingDTO;
 import uk.gov.justice.laa.maat.scheduled.tasks.entity.RepOrderBillingEntity;
+import org.apache.commons.lang3.StringUtils;
 
 @Slf4j
 public class RepOrderBillingMapper {
     private RepOrderBillingMapper() { }
 
     public static RepOrderBillingDTO mapEntityToDTO(RepOrderBillingEntity repOrderEntity) {
-        Integer magsCourtId = !StringUtils.hasLength(repOrderEntity.getMagsCourtId())
+        Integer magsCourtId = StringUtils.isEmpty(repOrderEntity.getMagsCourtId())
```

Comment:
As a heads-up, I changed this to use `hasLength` in my recent PR adding the mapper in, from `isEmpty`, as the latter is now deprecated and the docs recommended switching to `hasLength`:

<img width="480" height="433" alt="Screenshot 2025-07-31 at 15 53 26" src="https://github.com/user-attachments/assets/3f518ce3-198e-437a-a203-057ce6dcd040" />